### ISSUE TITLE
Can't sync HBO Max

### ISSUE THREAD

**Issue by joshsalvi** [trakt-tools/universal-trakt-scrobbler#280]
The settings keep stating "Not Logged In: Looks like you're not logged in to this service. Please go to the page below and log in before trying to sync." 

I go to the link, log in to HBO Max, and the error still appears.

I've also tried signing out of all devices, and still the error appears.

What does happen, however, is that although any new tab I open will be logged in to HBO Max, clicking the link takes me to a landing page where I am NOT logged in. 

Is there perhaps a setting in HBO Max that I need to change to make this work? Or within the extension?

**Comment by hugovk** [comment]
Same here, Chrome 109.0.5414.119 on macOS.


> **Not Logged In**
>
> Looks like you're not logged in to this service. Please go to the page below and log in before trying to sync.
>
> `https://www.hbomax.com/`


When I click `https://www.hbomax.com/` I'm redirected to the `https://www.hbomax.com/fi/fi` landing page, that doesn't show any logged-in status.

From there, if I click the "Sign in" button, that goes to `https://play.hbomax.com/signIn` and redirects to `https://play.hbomax.com/page/urn:hbo:page:home` where I'm already signed in.

**Comment by hugovk** [comment]
It's working now, although it's only showing history back to 5th Feb 2023 (so 1 month and 1 day). 

Is it possible to fetch more history, like with Netflix? Edit: history issue already reported at https://github.com/trakt-tools/universal-trakt-scrobbler/issues/245

**Comment by dannydickson** [comment]
Yeah, seems like this could be added to the "Known Issues" in the readme? There are several open issues showing the same thing.

**Comment by dhowell66** [comment]
Are other people still able to sync now that HBO Max is now Max? I haven't been able to sync since the changeover.

**Comment by motamedn** [comment]
> Are other people still able to sync now that HBO Max is now Max? I haven't been able to sync since the changeover.

I haven't been able to sync since they changed HBO Max to Max either.

**Comment by tlauen** [comment]
> It's working now, although it's only showing history back to 5th Feb 2023 (so 1 month and 1 day).
> 
> Is it possible to fetch more history, like with Netflix? Edit: history issue already reported at #245

How did you get it to work? I'm having the same issue I think. I'm logged in, but the plugins says I'm not...

**Comment by hugovk** [comment]
@tlauen I don't think I did anything, but don't have HBO right now.

**Comment by dhowell66** [comment]
So now that Max is back to HBO Max, I tried to sync history and it doesn't work. It gives the following error:

"Looks like you're not logged in to this service. Please go to the page below and log in before trying to sync.

https://www.hbomax.com/"

Even though I am definitely logged in.
